core/frontend/src/IModelConnection.ts

```diff
@@ -460,6 +476,19 @@ export abstract class IModelConnection extends IModel {
     return (this.noGcsDefined ? this.spatialToCartographicFromEcef(spatial, result) : this.spatialToCartographicFromGcs(spatial, result));
   }
 
+  /** Convert a point in this iModel's Spatial coordinates to a [[Cartographic]] using the Geographic location services for this IModelConnection or [[IModel.ecefLocation]].
+ * @param spatial A point in the iModel's spatial coordinates
+ * @param result If defined, use this for output
+ * @returns A Cartographic location (WGS84 horizontal datum)
+ * @throws IModelError if [[isGeoLocated]] is false or point could not be converted.
+ * @see [[spatialToCartographicFromGcs]]
+ * @see [[spatialToCartographicFromEcef]]
+ * @beta
+ */
+  public async spatialToWGS84Cartographic(spatial: XYAndZ, result?: Cartographic): Promise<Cartographic> {
```

Comment:
If a special method just for WGS84 is important, it should take an array not a single point.